tsconfig.json

```diff
@@ -16,6 +16,12 @@
       "@/*": ["./*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    "next.config.js",
+    "pages/hooks/query-keys.ts"
```

Comment:
Nope. Remove this.

Also, you should only have files/folders inside the `pages` directory that you want to translate to actual routes. We had this conversation during the weekend.

`/pages/hooks/query-keys.ts` tells Next that it should prepare a route with `/hooks/query-keys`, which I am confident that you do not want.